plugins/org.wso2.developerstudio.eclipse.artifact.dataservice/META-INF/MANIFEST.MF

```diff
@@ -19,6 +20,7 @@ Import-Package: org.eclipse.jdt.core,
  org.wso2.developerstudio.eclipse.capp.maven.utils;version="[6.0.0,7.0.0)",
  org.wso2.developerstudio.eclipse.ds.presentation.ui,
  org.wso2.developerstudio.eclipse.general.project.dialogs;version="[6.0.0,7.0.0)",
+ org.wso2.developerstudio.eclipse.gmf.esb.presentation,
```

Comment:
Can we remove the esb dependency for the dss side?

Reply:
Fixed